bigframes/series.py

```diff
@@ -1809,14 +1841,17 @@ def to_markdown(
         buf: typing.IO[str] | None = None,
         mode: str = "wt",
         index: bool = True,
+        allow_large_results=None,
```

Comment:
Nit: we don't want people to access allow_large_results positionally.

```suggestion
        *,
        allow_large_results=None,
```

Reply:
Updated.